core/src/main/java/io/confluent/kafka/schemaregistry/metrics/SchemaRegistryMetric.java

```diff
@@ -16,31 +16,50 @@
 package io.confluent.kafka.schemaregistry.metrics;
 
 import org.apache.kafka.common.MetricName;
+import org.apache.kafka.common.metrics.MeasurableStat;
 import org.apache.kafka.common.metrics.Metrics;
 import org.apache.kafka.common.metrics.Sensor;
 import org.apache.kafka.common.metrics.stats.Value;
 
 import java.util.concurrent.atomic.AtomicLong;
 
 public class SchemaRegistryMetric {
+  @Deprecated
   private final AtomicLong count = new AtomicLong();
   private final Sensor sensor;
 
+  @Deprecated
   public SchemaRegistryMetric(Metrics metrics, String sensorName, MetricName metricName) {
+    this(metrics, sensorName, metricName, new Value());
+  }
+
+  public SchemaRegistryMetric(Metrics metrics, String sensorName, MetricName metricName,
+                              MeasurableStat measurableStat) {
     sensor = metrics.sensor(sensorName);
-    sensor.add(metricName, new Value());
+    sensor.add(metricName, measurableStat);
   }
 
+  @Deprecated
   public void increment() {
     sensor.record(count.addAndGet(1));
   }
 
+  @Deprecated
   public void set(long value) {
     count.set(value);
     sensor.record(value);
   }
 
+  @Deprecated
   public long get() {
     return count.get();
   }
+
+  public void record() {
+    sensor.record();
```

Comment:
This will just record 1.0L, right? 
Should we add a comment here, indicating the same?